service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java

```diff
@@ -37,9 +41,11 @@ private void performVerification(final String caseId, final CaseDetails caseDeta
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, String.format("No case found with id %s", caseId));
         }
 
-        if (!caseDetails.getState().equals("Submitted")) {
+        List<String> donNotSendStates = asList("Open", "Deleted");
```

Comment:
can be static, List.of is preferable (produces immutable list which should be used here)